controllers/constant/odlm.go

```diff
@@ -777,8 +777,10 @@ spec:
                 - command:
                   - bash
                   - '-c'
-                  - >-
+                  args:
+                  - |
                     kubectl delete pods -l app.kubernetes.io/name=cloud-native-postgresql
```

Comment:
could this error and exit in case there is not yet running edb operator Pod?

Reply:
I think `deleting pod by label` will not return error in case edb operator pod is not found.
```
➜  ~ oc delete pod -l app.kubernetes.io/name=cloud-native-postgresql -n test-edb
pod "postgresql-operator-controller-manager-1-18-7-6b66dcccf-xrjwt" deleted
➜  ~ echo $?
0

➜  ~ oc delete pod -l app.kubernetes.io/name=xxxxxxxxxx -n test-edb
No resources found
➜  ~ echo $?
0
```

But indeed if we try to delete a specific pod, it returns error in case that pod does not exist.
```
➜  ~ oc delete pod xxxxxx
Error from server (NotFound): pods "xxxxxx" not found
➜  ~ echo $?
1
```